actionview/Rakefile

```
@@ -101,7 +92,8 @@ end
 namespace :ujs do
   desc "Starts the test server"
   task :server do
-    system "bundle exec rackup test/ujs/config.ru -p 4567 -s puma"
+    spawn("bundle exec rackup test/ujs/config.ru -p 4567 -s puma")
+    system("npm test -- --no-single-run --browsers Chrome")
```

I suggest we prefer explicit argument lists. It should also be a little faster IIUC it does not need to spawn a shell to parse the command.

```suggestion
    spawn("bundle", "exec", "rackup", "test/ujs/config.ru", "-p", "4567", "-s", "puma")
    system("npm", "test", "--", "--no-single-run", "--browsers", "Chrome")
```

---

In addition, are we hard coding `puma`? Is there a reason for that?

Strictly speaking, `rackup` doesn't exist in Rails 3 (it's now in the `rackup` gem). I wonder if we should be using `bin/rails s` or something similar?